assets/javascripts/discourse/components/post-voting-comment-composer.js

```diff
@@ -0,0 +1,31 @@
+import { action } from "@ember/object";
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+
+export default class PostVotingCommentComposer extends Component {
+  @service siteSettings;
+
+  @tracked value = this.args.raw ?? "";
+
+  @action
+  onInput(event) {
+    this.value = event.target.value;
+    this.args.onInput?.(event.target.value);
+  }
+
+  get isValueTooShort() {
+    return (
+      this.value.length > 0 &&
+      this.value.length < this.siteSettings.min_post_length
```

Comment:
I'm not convinced, if it is worth bothering with this edge case here, but what if `this.siteSettings.min_post_length` > `this.siteSettings.qa_comment_max_raw_length`?

Reply:
I don't think it's worth it, `qa_comment_max_raw_length` is a hidden setting with a default of 600 (not sure why it's hidden though) and the error messages are a good hint on how to "solve" it if it happens